homeassistant/components/media_player/sonos.py

```diff
@@ -38,12 +38,23 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """ Sets up the Sonos platform. """
     import soco
+    import socket
 
     if discovery_info:
         add_devices([SonosDevice(hass, soco.SoCo(discovery_info))])
         return True
 
-    players = soco.discover()
+    players = None
+    hosts = config.get('hosts', None)
+    if hosts:
+        players = []
+        for host in hosts.split(","):
+            host = socket.gethostbyname(host)
+            _LOGGER.warning("Sonos host %s" % host)
```

Comment:
Sry it was an debug print left unintentionally.
No other comments?

/R

On Tue, Jan 19, 2016, 20:43 Paulus Schoutsen <notifications@github.com>
wrote:

> In homeassistant/components/media_player/sonos.py
> <https://github.com/balloob/home-assistant/pull/937#discussion_r50163948>:
>
> >
> >      if discovery_info:
> >          add_devices([SonosDevice(hass, soco.SoCo(discovery_info))])
> >          return True
> >
> > -    players = soco.discover()
> > +    players = None
> > +    hosts = config.get('hosts', None)
> > +    if hosts:
> > +        players = []
> > +        for host in hosts.split(","):
> > +            host = socket.gethostbyname(host)
> > +            _LOGGER.warning("Sonos host %s" % host)
>
> Warning should only be used for non-fatal errors.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/937/files#r50163948>.
>